packages/7-todo-tdd/src/components/todo-app/TodoApp.test.tsx

```diff
@@ -0,0 +1,157 @@
+import { ReactWrapper } from 'enzyme';
+import { wrapperGenerator } from '../../styles/utils';
+import { TodoAppDriver } from './TodoApp.test.driver';
+import { s } from './TodoApp';
+import { Chance } from 'chance';
+import { TodoItemDriver } from '../todo-item/TodoItem.test.driver';
+import { todosService } from '../../services/TodoService';
+import { ITodoDTO } from 'fed-todo-journey_todo-common';
+
+const chance = new Chance();
+const h = wrapperGenerator('.');
+
+
+describe('app main components should be rendered', () => {
+    let mountedTodoApp: ReactWrapper;
+
+    beforeEach(async () => {
+        const todosDriver: TodoAppDriver = new TodoAppDriver();
+        await todosDriver.waitForAppToUpdate();
+        mountedTodoApp = todosDriver.getAppComponent();
+    });
+
+    it('app should be rendered', () => {
```

Comment:
the way we read tests is "it...." so a nicer way of writing it would be:
`it('should render the app');`

even though it doesn't really describe what this test does. what is the value in this test. this might be one of these tests that were needed while writing the code but now can be removed.

Reply:
got it